### DISCUSSION TITLE
How to allow free rotation vertically (up/down)

### DISCUSSION THREAD

**Discussion by tnageleweb** [google/model-viewer#4970] [Q&A] [answered]
Hi,
Is there a settings to allow drag up/down to behave like dragging right/left. I tried almost all camera-Orbit settings but it seems there is restriction from source code. Appreciate any help, thank you.

**Comment by elalish** [comment, marked as answer]
No, we intentionally restrict the model from going upside-down, as this is confusing to users and ends up reversing the apparent left-right motion.

**Reply by tnageleweb** [reply]
Thanks very much.

**Comment by RajatDBazaar** [comment]
@tnageleweb did you got any other way to do it I am also facing the same issue please help me if you have find any way to do it??
